packages/fast-components-react-msft/src/action-trigger/examples.data.tsx

```diff
@@ -81,6 +81,15 @@ const examples: ComponentFactoryExample<ActionTriggerProps> = {
             },
             "data-sketch-symbol": "Action trigger - stealth",
         } as any,
+        {
+            appearance: ActionTriggerAppearance.stealth,
+            href: testDestination,
+            glyph: {
+                id: svgSchema.id,
+                props: {},
+            },
+            "data-sketch-symbol": "Action trigger - stealth",
```

Comment:
We don't really seem to use/support this anymore, but the Sketch symbol name should be different than above. Maybe something like "Action trigger - stealth - icon-only".